working/2364 - primary constructors/feature-specification.md

```diff
@@ -439,15 +496,15 @@ class B extends A { // OK.
 class B(int a) extends A(a); // Could be supported, but isn't!
 ```
 
-There are several reasons why this is not supported. First, primary constructors
-should be small and easy to read. Next, it is not obvious how the
-superconstructor arguments would fit into a mixin application (e.g., when the
-superclass is `A with M1, M2`), or how readable it would be if the
-superconstructor is named (`class B(int a) extends A.name(a);`). For instance,
-would it be obvious to all readers that the superclass is `A` and not `A.name`,
-and that all other constructors than the primary constructor will ignore the
-implied superinitialization `super.name(a)` and do their own thing (which might
-be implicit).
+There are several reasons why this is not supported. First, primary
+constructors should be small and easy to read. Next, it is not obvious how
+the superconstructor arguments would fit into a mixin application (e.g.,
+when the superclass is `A with M1, M2`), or how readable it would be if the
+superconstructor is named (`class B(int a) extends A.name(a);`). For
```

Comment:
(I think it's fine.
```dart
class BananaSouffle(
   final int spoons,
   final Color sprinkles,
   BananaSort sort
) extends Banana(sort) with Edible;
```
So, yes, I'll argue that. It will make the syntax useful for exactly the kind of simple classes 
that just extend their superclass with extra fields.
```dart
class Person(
  final String name, 
  final int age, {
  super.id
}) extends LegalEntity(kind: Entity.person);
```
)

Reply:
Findes det virkelig, banansoufflé?! ;-)

In any case, we could do it for non-mixin-application superclasses.